### DISCUSSION TITLE
Business 20250208 VIS photography

### DISCUSSION THREAD

**Discussion by Cheng-ChiAn** [J-tour/J-tour.github.io#30] [General]
目前預計是2/8或2/9的下午 2個小時拍攝
地點是在新竹市找練團室
1:攝影師推薦加上街拍問我們是否接受?
2:攝影師想了解我們的成團故事 樂團簡介 成員介紹 表演內容風格
  我們是否有資料?
3:預算問題 兩小時拍攝我們預算多少? 讓我心裡有個底(目前都還沒談到價格XD)

4:拍攝服裝問題 攝影師需要確認我們當天的服裝

歡迎就各別問題留言討論~

**Comment by Cheng-ChiAn** [comment]
1.我ok
2.可能每個人寫自己簡短自介? 然後之前昊昊架的網站上資料可以用?
3. 攝影師會有兩人前來 我估計在4000~6000之間 再請各位評估

**Reply by ShengHaoLo** [reply]
感謝麒安整理這麼多資料，下次會議的時候可以來討論看看接下來的AR 🥅

**Comment by DipsyKoo** [comment]
1:我可以～
2:我們好像還沒有很明確的風格，我連自己都不知道怎麼介紹🤣
3:預算交給你們討論～
4:我常見的那些穿著都可以用來拍～

**Comment by xinyannn1** [comment]
2/8好像要補班！！想優先選擇2/9

1:街拍OK

2:稍微發想一下好了🤔
【成團故事】
一群人，因熱愛音樂而相識，因喜歡分享而走上街頭，之於他們，街頭不僅是展演的舞臺，更是與世界對話的方式，透過音樂傳遞愛與正能量，陪伴在每個追夢人的身旁，在每個街角。

【內容風格】
表演內容以不插電的形式為主，藉由翻唱流行、民謠歌曲分享音樂與真摯的情感。團體成員們沒有固定的位置與角色，組合方式多元而靈活，增添演出的變化與趣味。

3:不太了解行情，但感覺3,000-6,000差不多。

4:服裝可以看大家需不需要配色，我都OK。

**Comment by ShengHaoLo** [comment]
希望可以在 2025/2/9，新竹的話沒問題，錄音室我依稀記得10年前是500NT/h，不知道現在價格有沒有調整。

1. OK
2. 我之前架的網站上的資料可以用，不過最近一次跟[忻妍](@xinyannn1 )開會討論有打算改用[Canva](https://www.canva.com)讓忻妍比較好編輯，也有請忻妍在Canva整理品牌形象到白板，也歡迎大家[加入Canva 一起共同編輯](https://www.canva.com/brand/join?token=-QXk3IxWvzQy9gi6GRLDtQ&brandingVariant=edu&referrer=team-invite)，我先把之前的資料稍微整理如下：

- 🌱 **品牌故事**
J-tour 正式成立於 2023 年 3 月。起初是一群喜歡在街頭營造驚喜的朋友們，一起玩音樂、成為街頭藝人，接著再用音樂與故事，邂逅路過的朋友們，成立至今已在北、桃、竹大大小小地表演許多場專場、街頭以及非營利活動。
- 👥 **經營團隊**
除了專場表演外，J-tour 秉持著文化近用權與公共藝術的精神，每十場表演至少會有一場街頭表演，並且保障街頭表演所用經費需達營運經費的百分之一，以確保在地性、公共性、藝術性的活動能取之於社會，用之於社會，讓表演成為真正的「公共財」。
- 🧑🏼‍🤝‍🧑🏼 **成員介紹**
昊昊：浪漫知性的全方位音樂人，縝密地計算音符下個位置會出現的位置。
迪西：絕世美聲，擅長透過溫暖嘹亮的聲線感動人心，傳遞正念療育群眾。
忻妍：清新脫俗的鄰家小孩，撥弦、鍵盤、唱作俱佳，與原聲樂器最合拍。
Home7：神秘的撥弦樂器演奏家，彈指之間通透著合成器的音色所適配的指法。
Sean：彈唱得俏皮地遊走在嘻哈與抒情樂間，總能在模進上帶來弦外之音。
安安：真摯的吟遊詩人，訴說著都會、城市、田野間市井小民大大小小的故事。
- 🎭 **表演風格**
表演特色是表演者沒有固定的表演樂器配置，時常會根據表演內容交換位置，並秉持著高度與觀眾互動的頻率，即便觀眾要加入表演，也能即時響應。音樂風格橫跨古典、爵士、流行、搖滾，從說唱到演奏樂、原聲樂器到合成器、翻唱到創作皆有沾染，是謂街頭音樂的變色龍，完美地融入當下的情境中。

3. 如果有外拍，通常 2 小時拍不完，可以直接算 5,000 NT/半天 或 8,000 NT/天。
4. 服裝以平時表演會出現的服裝為主，建議表演者們多備 2 套，整體色調不要太暗就好。

**Comment by Cheng-ChiAn** [comment]
目前報價是5000 請保密XDD
攝影師想拍小短劇 類似廣告


![IMG_2580](https://github.com/user-attachments/assets/800c10ff-c462-403d-abf3-88707e6d2b9f)
![IMG_2581](https://github.com/user-attachments/assets/e3655012-3af7-4e55-a200-dfb84b5397b4)
![IMG_2582](https://github.com/user-attachments/assets/665f7fdb-dc83-4463-be4f-91b69bb7cc74)
![IMG_2583](https://github.com/user-attachments/assets/c43a2149-a203-4e88-8c93-42f4efed05bd)
![IMG_2589](https://github.com/user-attachments/assets/f74b7b52-b5bc-4d8d-8647-288fa99b08ab)
